### DISCUSSION TITLE
How to configure for custom css classes which add multiple style props

### DISCUSSION THREAD

**Discussion by neel1998** [dcastil/tailwind-merge#327] [Help]
I have many custom typography classes like,

```
.typography-h1 {
   font-size: 1.8rem;
   line-height: 2.0rem;
   font-weight: 600
}
```

is there a way to configure twMerge so that when I do something like `twMerge('typography-h1 font-700')` the resultant styles have font-weight of 700

**Comment by dcastil** [comment]
Hey @neel1998! 👋

Yes, you can do it. I wrote a little tutorial to a similar case here: https://github.com/dcastil/tailwind-merge/discussions/323#discussioncomment-7336816

Let me know if this helps!
